modules/fx/arrow-fx-rx2/src/main/kotlin/arrow/fx/rx2/extensions/observablek.kt

```diff
@@ -303,11 +286,9 @@ interface ObservableKFunctorFilter : FunctorFilter<ForObservableK>, ObservableKF
     fix().filterMap(f)
 }
 
-@extension
-interface ObservableKMonadFilter : MonadFilter<ForObservableK>, ObservableKMonad {
-  override fun <A> empty(): ObservableK<A> =
-    Observable.empty<A>().k()
-
-  override fun <A, B> Kind<ForObservableK, A>.filterMap(f: (A) -> Option<B>): ObservableK<B> =
-    fix().filterMap(f)
-}
+// TODO Could be revisited in the future. Right now it fails to pass the laws due to the notion of errors.
```

Comment:
This should be just removed since we are not deprecating it. Observable will never be able to support this with the current encoding we have for MTL based on subtypes.